packages/TorneloScoresheet/src/context/AppModeStateContext.tsx

```diff
@@ -1,48 +1,17 @@
 import React, { useContext, useState } from 'react';
-import { makegoToTablePairingSelection as makeEnterPgnToPairingSelection } from '../goToTablePairingSelection';
 import { AppModeState, AppMode } from '../types/AppModeState';
-import { GameInfo } from '../types/chessGameInfo';
-import { Result } from '../types/Result';
 
 // The global state for the app
-const AppModeStateContext = React.createContext<
+export const AppModeStateContext = React.createContext<
   [AppModeState, React.Dispatch<React.SetStateAction<AppModeState>>]
 >([{ mode: AppMode.EnterPgn }, () => undefined]);
 
```

Comment:
I think we should either not export this, or document it saying that it should not be imported out side of this directory. Otherwise, down the track (even after we're done maintaining the project), somebody might import this to a random view and mutate the state directly.

This is why we hade the `makeEnterTablePgnToPairingSelection` function that takes the setter as an argument - it means that we can give that specific function the ability to mutate the global state without exporting it globally.

Ideally we'd have all of the transition functions in this function so that there is no way to leak the `setAppModeState` function, but I think that passing it as a parameter, rather than exporting it is the best compromise.

There might be a better way though, let me know what you think